lib/src/extensions.dart

```diff
@@ -0,0 +1,6 @@
+import 'characters.dart';
+
+extension FromString on String {
```

Comment:
I'd say `StringCharacters` as name. Then an explicit use would look like `StringCharacters(string).characters, not `FromString(string).characters`.
The latter isn't bad, I just like the former more.